.travis.yml

```diff
@@ -118,6 +118,10 @@ before_install:
     fi
 install:
   - if [[ "${TEST_SAGE}" != "true" ]]; then
```

Comment:
I am not able to figure how to do that.
If you set `TEST_SAGE="true"` in the `env` of each version, the test wouldn't run at all.